### PR TITLE
Test installing Flynn from a TUF repository

### PR DESCRIPTION
**Pull request by lmars**
Incremental patch for #617.

### CODE REVIEW

**Comment by titanous** [comment]
Looks like https://github.com/flynn/flynn/commit/adc88622ce7cb2815a8637b2a1b49da4ab248279 and https://github.com/flynn/flynn/commit/7795f3ccf672adcc7540e5adf75d3cd9ef43177c could be merged into master directly?

File: test/test_release.go [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+export GIT_COMMIT=dev
+export GIT_BRANCH=dev
+export GIT_TAG="v20150131.0-test"
+export GIT_DIRTY=false
+
+# send all output to stderr so only version.json is output to stdout
+(
+  # tup hangs waiting on the FUSE socket after building the components, so
+  # for now we just kill the connection ater an appropriate amount of time
+  (sleep 30; sudo bash -c 'echo 1 > /sys/fs/fuse/connections/*/abort') &
```
Comment (titanous) [inline test/test_release.go]: Please file an issue in our tracker with details about this issue and reference the issue from the comment.

Comment (titanous) [inline test/test_release.go]: Also, dumb idea is to use inotify to watch for whatever the last file generated is.

Comment (lmars) [inline test/test_release.go]: inotify would be good but I am not 100% sure what to wait for, and given this test takes ~4mins per iteration, I am tempted to leave as is for now and see how it holds up in CI.

**Comment by lmars** [comment]
@titanous comments addressed

**Comment by titanous** [comment]
LGTM

**Pull request merged**